create.py

```diff
@@ -3,43 +3,79 @@
 import warnings
 
 import pymysql as mdb
+import sqlite3
 
 from tt import *
 
 warnings.filterwarnings('ignore')
 
 
+def create_table_command_mysql():
+    sql_commands = [('''
+    use {db};'.format(args_dict['db']);
+    create table if not exists {table} (
+        id int not null auto_increment,
+        date varchar(10),
+        start time(0),
+        end time(0),
+        project varchar(8),
+        primary key (id)
+    );
+    ;
+    '''.format(db=args_dict['db'], table=args_dict['table']))]
+
+    sql_commands += ['create database if not exists {}'.format(args_dict['db'])]
```

Comment:
As I was looking at this this is going to append the list with the `create database...` *after* the `use database...` call, which should result in an error. I think this goes to testing -- I can test this in mysql if you don't have mysql installed. LMK.